### PR TITLE
travis: PHP 7.0 nightly added + few improvements

### PR DESCRIPTION
**Pull request by TomasVotruba**
Do not download composer manually, use Travis' one.
Drop `--dev`, on by default for almost a year.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,14 +4,16 @@ php:
...
 matrix:
-    allow_failures:
-        - php: hhvm
+  allow_failures:
+    - php: hhvm
 
 before_script:
-  - curl -s http://getcomposer.org/installer | php
-  - php composer.phar install --dev
+  - composer self-update
```
Comment (GrahamCampbell) [inline .travis.yml]: :-1: Should be ommited.

File: .travis.yml [hunk 2]
```diff
@@ -4,14 +4,16 @@ php:
...
+  - composer install
```
Comment (GrahamCampbell) [inline .travis.yml]: You should have `--prefer-source` to prevent us from selfishly busting the api limits for the box.

File: .travis.yml [hunk 3]
```diff
@@ -4,14 +4,16 @@ php:
...
+  - 7.0
   - hhvm
 
 matrix:
-    allow_failures:
-        - php: hhvm
+  allow_failures:
+    - php: hhvm
 
 before_script:
```
Comment (GrahamCampbell) [inline .travis.yml]: should be `install`

**Comment by brunogaspar** [comment]
Should be fixed.

**Comment by TomasVotruba** [comment]
@brunogaspar Could you elaborate?

**Comment by GrahamCampbell** [comment]
https://github.com/cartalyst/sentinel/commit/5b1b8d0765c060aaef9715a8c64b7f72853c2771

**Comment by TomasVotruba** [comment]
@GrahamCampbell Thanks

@brunogaspar Commit/issue reference would be helpful, for me and for commit history.

**Comment by brunogaspar** [comment]
@TomasVotruba Sorry, i forgot to include the `Closes: #79` on the commit message.

**Comment by TomasVotruba** [comment]
@brunogaspar :+1:

**Pull request closed without merging**